src/Template/Layout/js/app/pages/modules/view.js

```diff
@@ -97,6 +98,28 @@ export default {
             document.body.appendChild(iframe.$el);
         },
 
+        async showFlashMessages() {
+            // fetch flash messages template
+            const messages = await fetch('/dashboard/messages');
```

Comment:
with the suggested change above there's no need to call `'/dashboard/messages'`
an error object can be passed to this method and displayed